### ISSUE TITLE
Testnet sync stuck

### ISSUE THREAD

**Issue by arrix** [CityOfZion/neo-python#539]
### Current behavior

- testnet sync stuck
```
neo> [I 180809 16:43:39 LevelDBBlockchain:854] Could not persist block argument out of range
Unhandled error in Deferred:

neo>
neo> state
Progress: 1684029 / 1684914
Block-cache length 670
Blocks since program start 31
Time elapsed 1.04607045 mins
Blocks per min 29.634715329163537
TPS: 1.3064766973072097

neo>







[TestNet] Progress: 1684030/1684916
```

### How to reproduce

- np-prompt
- if it gets stuck, quit and restart
- sync will be stuck after the error is shown: Could not persist block argument out of range

### Your environment

Let us know in what environment you're running into the issue:

- OS: Ubuntu 16.06
- neo-python version: 0.7.6
- Python version: 3.6.6

**Comment by tomascarvalho** [comment]
Related to #492

**Comment by localhuman** [comment]
This should be solved by #540
